lapis2/src/main/kotlin/org/genspectrum/lapis/controller/LapisController.kt

```diff
@@ -118,6 +128,43 @@ class LapisController(private val siloQueryModel: SiloQueryModel, private val re
             sequenceFilters.associate { it.key to it.value },
         )
     }
+
+    @GetMapping("/details")
+    @LapisDetailsResponse
+    fun details(
+        @Parameter(
+            schema = Schema(ref = "#/components/schemas/$SEQUENCE_FILTERS_SCHEMA"),
+            explode = Explode.TRUE,
+            style = ParameterStyle.FORM,
+        )
+        @RequestParam
+        sequenceFilters: Map<String, String>,
+        @Schema(description = DETAILS_FIELDS_DESCRIPTION)
+        @RequestParam(defaultValue = "")
+        fields: List<String>,
+    ): List<DetailsData> {
+        requestContext.filter = sequenceFilters
+
+        return siloQueryModel.getDetails(
+            sequenceFilters.filterKeys { !nonSequenceFilterFields.contains(it) },
+            fields,
+        )
+    }
+
+    @PostMapping("/details")
+    @LapisDetailsResponse
+    fun postDetails(
+        @Parameter(schema = Schema(ref = "#/components/schemas/$DETAILS_REQUEST_SCHEMA"))
+        @RequestBody()
+        request: SequenceFiltersRequestWithFields,
+    ): List<DetailsData> {
+        requestContext.filter = request.sequenceFilters
+
+        return siloQueryModel.getDetails(
+            request.sequenceFilters,
```

Comment:
Shouldn't this be `request.sequenceFilters.filterKeys { !nonSequenceFilterFields.contains(it) },` as for aggregrated? It seems that `fields` is also passed as a filter?

Reply:
Or is it not necessary above?

Reply:
Ah thanks, good catch.